Misc/NEWS.d/next/Documentation/2023-08-14-20-38-39.gh-issue-107957.DU3mID.rst

```diff
@@ -0,0 +1 @@
+Fix sample outputs for :func:`dir()` function in :mod:`enum` docs.
```

Comment:
According to my experience, there is no need to add NEWS when we only modify the document, just wait for the triage member to add the `skip news` label.